src/arrayMethodSort.js

```diff
@@ -6,6 +6,40 @@
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
```

Comment:
```suggestion
  [].__proto__.sort2 = function(compareFunction = () => {
    return something;
  }) {
```

you can do something like this